--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -795,6 +795,15 @@
       origin_trial_feature_name: "FeaturePolicyReporting",
       status: "experimental"
     },
+    {
+      name: "FingerprintingClientRectsNoise",
+    },
+    {
+      name: "FingerprintingCanvasMeasureTextNoise",
+    },
+    {
+      name: "FingerprintingCanvasImageDataNoise",
+    },
     {
       name: "FeaturePolicyVibrateFeature"
     },

